core/src/main/java/org/elasticsearch/search/suggest/term/TermSuggestionBuilder.java

```diff
@@ -457,9 +490,9 @@ public SortBy readFrom(final StreamInput in) throws IOException {
             return values()[ordinal];
         }
 
-        public static SortBy fromString(final String str) {
+        public static SortBy resolve(final String str) {
             Objects.requireNonNull(str, "Input string is null");
-            return valueOf(str.toUpperCase(Locale.ROOT));
+            return valueOf(str.toUpperCase(Locale.US));
```

Comment:
nit: we almost never use `Locale.US` exept for some number formating. While I think it doesn't make a difference for the enum names in question here, I'd suggest going with `Locale.ROOT`